infra/lib/attendant-ide/compute-construct.ts

```diff
@@ -129,7 +136,10 @@ export class ComputeConstruct extends Construct {
         `export PATH="$HOME/.local/share/fnm:$PATH"`,
         `eval "\`fnm env\`"`,
         `cd /home/${whoamiUser}/${workshopDirectory}`,
-        `npm ci`
+        `npm ci`,
+        `npm run utils:createConfig`,
+        `npm run frontend:build`,
+        `aws s3 cp --recursive ./frontend/build s3://${props.websiteBucketName}`
```

Comment:
I need to fix this:

Trying to find output file locally.
Unable to find output file locally, trying remotely.
Trying to find stack with PowerToolsWorkshop
Error: Unable to find stack among loaded ones
    at getStackName (file:///home/ec2-user/workshop/scripts/create-aws-exports.mjs:41:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async main (file:///home/ec2-user/workshop/scripts/create-aws-exports.mjs:136:21)
Unable to load CloudFormation stacks.
Did you run `npm run infra:deploy` in the project root?
file:///home/ec2-user/workshop/scripts/create-aws-exports.mjs:143
      throw new Error('Unable to find parameters locally or remotely.');